source/features/raw-view-link.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const createRawUrl = (href: string): string => {
+	const url = href.split('/');
+	url[3] = 'raw'; // Replaces 'blob'
+	return url.join('/');
+};
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		if (node instanceof Element) {
+			if (node.matches('js-diff-progressive-container')) {
+				observer.observe(node, {childList: true});
+			} else {
+				addRawBtn(node);
+			}
+		}
+	}
+});
+
+function addRawBtn(node: Element | null) {
+	if (node) {
+		for (const link of select.all('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']', node)) {
+			const href = link.getAttribute('href');
+			if (href) {
```

Comment:
We already select only links with `href`, so it always there

Reply:
It is but after enabling TS strict mode, `getAttribute` returns string or null. This is solved by getting the pathname as mentioned above.

Reply:
Like in the other comment, the null TS issue can be shut up with `!`, so

```ts
const href = link.getAttribute('href')!;
```

And `href` will always be `string` instead of `string | null`